src/App.css

```diff
@@ -0,0 +1,26 @@
+.App {
+  /* text-align: center; */
+  min-width: 100%;
+}
+
+* {
```

Comment:
We should probably remove all universal and tag selectors from our css and make sure that all the class selectors are super specific so they don't collide with the CSS rules in the Squarespace app

Reply:
GREAT catch. Was not thinking that far ahead. I'll update the selectors/classes accordingly.